website_sale_hide_price/models/website.py

```diff
@@ -18,4 +23,8 @@ class Website(models.Model):
 
     def _compute_website_show_price(self):
         for rec in self:
-            rec.website_show_price = request.env.user.partner_id.website_show_price
+            rec.website_show_price = (
+                False
+                if rec.website_hide_price
+                else request.env.user.partner_id.website_show_price
+            )
```

Comment:
suggestion: easier to read:
```suggestion
            rec.website_show_price = not rec.website_hide_price or request.env.user.partner_id.website_show_price
```

Reply:
Modified it.
Thanks